src/truffle/.gitignore

```diff
@@ -0,0 +1,2 @@
+build/contracts
+contracts/Hello.sol
```

Comment:
Instead of having `Hello.sol` here, let's come up with a more general purpose solution, like for example keeping your personal stuff in a `src/truffle/local` dir or something.